pkg/logqlmodel/logqlmodel.go

```diff
@@ -5,8 +5,8 @@ import (
 
 	"github.com/grafana/loki/pkg/querier/queryrange/queryrangebase/definitions"
 
-	"github.com/grafana/loki/pkg/logproto"
 	"github.com/grafana/loki/pkg/logqlmodel/stats"
+	"github.com/grafana/loki/pkg/push"
```

Comment:
Drive-by refactoring or intentional? IIRC we had to reference `logproto` because of some external dependency issue

Reply:
`github.com/grafana/loki/pkg/push` is its own package. I think it's actually better this way. `logproto` had a alias for `push`.

Reply:
right, it shouldn't break external usage of `github.com/grafana/loki/pkg/push` package, thank you for head up @dannykopping!